### PR TITLE
Issue #11 - Enhance the Issuer section to allow clicking through to issuing certificate tab

### PR DESCRIPTION
**Pull request by MauAraujo**
I added the expected functionality as observed in the image below: 
issuers
@april @glawrence, I would like some feedback on the styling and general implementation. 
Maybe in the future some animations could be added to make it friendlier to the user. Thanks in advance.

### CODE REVIEW

**Comment by glawrence** [comment]
Thanks @MauAraujo that looks great and is exactly what I had in mind, nice job!

The only minor comment would be whether you could turn the link off for Issuer Section in the Root CA tab, of course you will need to detect it is a built-in root but that is already being done as the tab title has "(built-in root)" on the end.

**Comment by april** [comment]
I agree with @glawrence about removing the link on the root CA (since it is self-signed).

As for the styles, I'm not 100% certain since that's a bit outside of my wheelhouse. I'm fairly sure that it shouldn't have a different "visited" color, and it's a bit abrupt that it's transitioning tabs. Let me see if I can get someone from the Firefox UX team to comment on this.

**Comment by MauAraujo** [comment]
@april @glawrence I removed the link on the root CA, and I also fixed the style so that the link is always the same color, even when it has been previously visited. I agree that the transitioning is rather abrupt. If it is OK, I can start looking into adding some light animations to make it look smoother and friendlier to the user. Thanks in advance for your feedback.
noroot

**Comment by glawrence** [comment]
This looks great now: keeping the links blue and not making the root CA issuer a link

However, it looks like you have pulled in some code from #42 which is a little confusing, probably better to keep these separate

**Comment by MauAraujo** [comment]
Sorry, that wasn't supposed to be there, it was a mistake. I fixed it now to only show the relevant code.

**Comment by MauAraujo** [comment]
After testing the code again, I noticed that when a certificate had more than 3 tabs, the tabbing was not working properly. I fixed it and now it works properly when there are multiple tabs. 
multipletabs

**Comment by glawrence** [comment]
I can confirm the latest code works in terms of the Root CA and the 4 tabs that show at https://www.hs-bremerhaven.de/start/ also work, so this is looking great, nice work! @april I think you could merge this, unless you want some tab change animation, but that could go in a new issue, maybe?

**Comment by april** [comment]
Hmm, I'm using the currently released public version of Certainly Something and it works fine with four tiers of certificates. Can you help me understand what you mean about it not working with four tabs?

**Comment by MauAraujo** [comment]
Yes, sorry if I wasn't clear before. After I reviewed my changes, I discovered that the links for switching tabs between issuers did not work properly when the certificate had more than 3 issuer tabs. This was because previously I hadn't considered the case, but I fixed my code in the last commit so that it works regardless of the number of issuer tabs.

File: src/viewer/js/render.js [hunk 1]
```diff
@@ -24,6 +24,7 @@ const postRender = () => {
   const buttons = document.getElementById('certificates').getElementsByClassName('panel-section-tabs-button');
   const certificates = document.getElementsByClassName('certificate');
   const longhexes = document.getElementsByClassName('long-hex');
+  const issuerlink = document.getElementsByClassName('issuer-link');
```
Comment (april) [inline src/viewer/js/render.js]: This should probably be pluralized, since there is more than one.

File: src/viewer/index.handlebars [hunk 2]
```diff
@@ -43,7 +43,7 @@
         {{else}}
           <div class="panel-section" id="panel-section-tabs">
             {{#each certs}}
-            <div class="panel-section-tabs-button {{#if @first}}selected{{/if}}" data-certificate-index="{{@index}}">{{ this.subject.cn }} {{#if this.isBuiltInRoot}}(built-in root){{/if}}</div>
+              <div class="panel-section-tabs-button {{#if @first}}selected{{/if}}" data-certificate-index="{{@index}}">{{ this.subject.cn }} {{#if this.isBuiltInRoot}}(built-in root){{/if}}</div>
```
Comment (april) [inline src/viewer/index.handlebars]: Why was this line changed?

File: src/viewer/index.handlebars [hunk 3]
```diff
@@ -69,8 +69,16 @@
           </div>
           {{#each this.issuer.entries}}
           <div class="panel-list-item">
-            <div class="text">{{lookup this 0}}</div>
-            <div class="text">{{lookup this 1}}</div>
+              <div class="text">{{lookup this 0}}</div>
+              {{#if (eq (lookup this 0)  "Common Name")}}
```
Comment (april) [inline src/viewer/index.handlebars]: You probably want to link the last entry, since Common Name will eventually be localized.

File: src/viewer/js/render.js [hunk 4]
```diff
@@ -47,6 +48,18 @@ const postRender = () => {
     });
   }
 
+    // setup event handler for issuer navigation
+    for (let node of issuerlink) {
```
Comment (april) [inline src/viewer/js/render.js]: This seems very complicated. The buttons are already numbered (via `data-certificate-index`), and it seems as if it would be better to find the current index and then click the button that has an index that is +1.

File: src/viewer/index.handlebars (hunk 2 above)
Comment (MauAraujo) [reply src/viewer/index.handlebars]: Sorry, adding that white space was unnecessary.

**Comment by glawrence** [comment]
I have tested the latest changes and it works for me, so that's a "Test: Passed" from me

**Comment by glawrence** [comment]
Sorry @april but how do you feel about merging this one, I can't see anything outstanding

**Comment by MauAraujo** [comment]
Is there anything that is missing? Should I proceed with adding animations? I honestly thought that was better as a separate issue, so I was waiting on this one to be merged. Please advise on what to so next.

**Comment by glawrence** [comment]
I think animation should be a separate issue, allows this to be closed off and also means another issue/PR can discuss just animation
